### DISCUSSION TITLE
Presto cluster is getting slower when running heavy for long time

### DISCUSSION THREAD

**Discussion by njalan** [trinodb/trino#13676] [General]
My version is prestosql 340. I have set hardConcurrencyLimit as 40. 

After the cluster is restarted,At the beginning most of the query can be finished within 30 seconds but after continue running 40 queries for one hour many queries are getting timeout(more then 120 seconds). But the query is the same queries are almost same as the queries running at the beginning.

**Comment by tooptoop4** [comment]
do u do queries with regex function? maybe 389 release helps https://github.com/trinodb/trino/issues/6405#issuecomment-1225040220
